starknet_py/tests/e2e/block_test.py

```diff
@@ -66,7 +66,9 @@ async def test_get_block_with_txs_pending(account):
 
 # TODO (#1419): Fix contract redeclaration
 @pytest.mark.asyncio
-async def test_get_block_with_txs_latest(account):
+async def test_get_block_with_txs_latest(account, map_class_hash):
+    # pylint: disable=unused-argument
+
```

Comment:
Param `map_class_hash` was added in purpose to create a new block - test setup executes some transactions that cause block creation